packages/roosterjs-content-model/test/domToModel/processors/listItemProcessorTest.ts

```diff
@@ -54,7 +54,7 @@ describe('listItemProcessor', () => {
                     blockType: 'BlockGroup',
                     blockGroupType: 'ListItem',
                     blocks: [],
-                    levels: [{ listType: 'UL', displayForDummyItem: 'block' }],
```

Comment:
This shows the code has regression. displayForDummyItem needs to be 'block' here